app/routers/timer.py

```diff
@@ -0,0 +1,13 @@
+from fastapi import APIRouter, Depends, Request
+from app.internal.timer import get_next_user_event_start_time
+from app.database.models import User
+from app.dependencies import get_db
+
+
+router = APIRouter()
+
+
+@router.get("/timer")
+def timer(request: Request, session=Depends(get_db)):
+    user = session.query(User).filter_by(id=1).first()
```

Comment:
This route queries the event of the user. It can make the system go really slow.

1. Prefer to use Depends(current_user) from Kobi's login system to get the user details without querying the database.
2. [Optional]: To make this commit merge-ready, add `last_added_event` to the localStorage. If it exists, take it from the localStorage and Voilà! You don't have to access the database (you don't even have to send requests to the server). If it doesn't, query and add it to the localStorage. Note that you need to update the localStorage if the user add a new event.

Reply:
1. Done :)
2. The countdown timer is counting till the current user's next event and not to the last added event.

Reply:
I understand it. The note means to veck if the new event becomes the last event. If it does, it should replace the next event in the localStorage